packages/libs/eda/src/lib/map/analysis/mapTypes/shared.tsx

```diff
@@ -304,6 +299,15 @@ export function useDistributionMarkerData(props: DistributionMarkerDataProps) {
     },
     enabled: overlayConfig != null,
```

Comment:
I wonder if this should be something like
```suggestion
    enabled: !overlayConfigResult.isFetching,
```

Reply:
Maybe `overlayConfig != null && !overlayConfigResult.isFetching`?

This would make sure the main marker data request doesn't get made with stale overlayConfig data perhaps?

What was your reasoning or hunch?

Reply:
Yes, that was my thinking.